.github/workflows/tests.yml

```diff
@@ -84,6 +84,14 @@ jobs:
         with:
           path: tardis-refdata/.git/lfs
           key: ${{ runner.os }}-lfs-${{ hashFiles('tardis-refdata/.lfs-assets-id') }}-v1
+      
+      - name: Clone tardis-sn/tardis-regressions
+        uses: actions/checkout@v4
+        with:
+          repository: atharva-2001/tardis-regressions
```

Comment:
When will this be changed to the "official" repository?

Reply:
I was thinking to change this once everything else is done.